nutty vendors an old and broken python2-only version of speedtest-cli.
Since it is already packaged (and python3 already), we can rely on it instead.

--

--- a/src/nutty.vala
+++ b/src/nutty.vala
@@ -1682,9 +1682,6 @@
 			speedtest_list_store.clear();
 			TreeIter iter;
 			if(shouldExecute){
-				if(! COMMAND_FOR_SPEED_TEST[0].contains(Constants.NUTTY_SCRIPT_PATH)){
-					COMMAND_FOR_SPEED_TEST[0] = Constants.NUTTY_SCRIPT_PATH+ "/" + COMMAND_FOR_SPEED_TEST[0];
-				}
 				execute_sync_multiarg_command_pipes(COMMAND_FOR_SPEED_TEST);
 				//handle unsucessfull command execution and raise error on infobar
 				if(!Utils.isExpectedOutputPresent(
--- a/data/scripts/meson.build
+++ b/data/scripts/meson.build
@@ -23,7 +23,6 @@
     'nutty_vnstat_script.sh',
     'nutty_traceroute_script.sh',
     'nutty_ports_script.sh',
-    'speedtest-cli'
 ]
 foreach a_script : script_names
     install_data(
